Development.podspec

```diff
@@ -89,6 +89,32 @@ Pod::Spec.new do |s|
     }
   end
 
+  s.app_spec 'Tutorial' do |app_spec|
+    app_spec.dependency 'BackStackContainer'
+    app_spec.dependency 'TutorialViews'
+    app_spec.dependency 'TutorialBase'
+    app_spec.dependency 'Tutorial1'
+    app_spec.dependency 'Tutorial2'
+    app_spec.dependency 'Tutorial3'
+    app_spec.dependency 'Tutorial4'
+    app_spec.dependency 'Tutorial5'
+    app_spec.source_files = 'swift/Samples/Tutorial/AppHost/Sources/*.swift'
+
+    app_spec.scheme = {
+      environment_variables: snapshot_test_env
+    }
+  end
+
+  s.test_spec 'Tutorial5Tests' do |test_spec|
+    test_spec.dependency 'BackStackContainer'
+    test_spec.dependency 'TutorialBase'
+    test_spec.dependency 'Tutorial5'
+    test_spec.dependency 'WorkflowTesting'
+    test_spec.requires_app_host = true
```

Comment:
can we also do `test_spec.app_host = 'Development/Tutorial'` so the tutorial app is the host? makes it easier to run the tests while working on the tutorial

Reply:
actually, the tutorial app might be in a weird state for compiling since its, ya know, a tutorial. probably better to use a generated host